# Removes the -fcomplete-member-pointers CFLAG, which was introduced in
# LLVM r333498 (i.e. after LLVM 6, the current stable at the time of writing)

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -743,14 +743,6 @@ config("compiler") {
     ldflags += [ "-Wl,--no-rosegment" ]
   }
 
-  # This flag enforces that member pointer base types are complete. It helps
-  # prevent us from running into problems in the Microsoft C++ ABI (see
-  # https://crbug.com/847724).
-  if (is_clang && !is_nacl && target_os != "chromeos" && !use_xcode_clang &&
-      (is_win || use_custom_libcxx)) {
-    cflags += [ "-fcomplete-member-pointers" ]
-  }
-
   # Pass the same C/C++ flags to the objective C/C++ compiler.
   cflags_objc += cflags_c
   cflags_objcc += cflags_cc
